### PR TITLE
Improve readme for manual build of Khiops windows installer

### LINKED ISSUE #445: Improve readme for manual build of Khiops windows installer
#### Description
Améliorations de détail sur la check-list du processus de build manuel

### Context
- Khiops version: 11

### PR DESCRIPTION
**Pull request by marcboulle**
Amélioration de détail
Pas nécessaire de faire un backport sur la V10

### CODE REVIEW

File: packaging/windows/nsis/README.md [hunk 1]
```diff
@@ -27,13 +27,19 @@ repository.
...
+%REM Before building the installer, update the following script arguments, mainly:
+%REM - KHIOPS_VERSION
+%REM - KHIOPS_REDUCED_VERSION
+%REM - KHIOPS_VIZ_INSTALLER_PATH, with last version of the installer available in the assets
+%REM - KHIOPS_COVIZ_INSTALLER_PATH, with last version of the installer available in the assets
+
 makensis ^
-   /DKHIOPS_VERSION=10.2.0-preview ^
+   /DKHIOPS_VERSION=10.2.0-rc.1 ^
    /DKHIOPS_REDUCED_VERSION=10.2.0 ^
```
Comment (popescu-v) [inline packaging/windows/nsis/README.md:46]: I'm not sure about how these versions should be updated, as they are moving targets according to the ongoing release. 

Perhaps, as this is a readme, we could follow either one of the two approaches listed below:
- we don't change anything to the version numbers themselves, but add a comment that these should be updated according to the pending release version
- we systematically update this to the version of the last release (e.g. 10.2.3 in our case).

Either of the two choices seems acceptable to me, although I would lean towards the first one, as it seems less expensive to me (it requires no further maintenance of the README upon each new release).

File: packaging/windows/nsis/README.md [hunk 2]
```diff
@@ -27,13 +27,19 @@ repository.
 
 ## How to build the installer manually
 1) Install NSIS and make sure `makensis` it is available in the `%PATH%`.
-2) Download and decompress the package assets to your machine.
-3) [Build Khiops in Release mode](https://github.com/KhiopsML/khiops/wiki/Building-Khiops)
+2) Download and decompress the package assets to your machine, on `packaging\windows\nsis\assets`
+3) [Build Khiops in Release mode](https://github.com/KhiopsML/khiops/wiki/Building-Khiops), to build the binaries and the jars
 4) In a console, go to the `packaging/windows/nsis` directory and execute
```
Comment (popescu-v) [inline packaging/windows/nsis/README.md]: I would also add somewhere that we need to position ourselves on the pending release Git tag (e.g. 10.2.3) before launching the package building process.

**Review by popescu-v — CHANGES_REQUESTED** [review]
There are 2 comments to address.

File: packaging/windows/nsis/README.md (hunk 1 above)
Comment (marcboulle) [reply packaging/windows/nsis/README.md:46]: J'ai juste changé "preview" en "rc.1" pour avoir un exemple réaliste de version officiel (à ma connaissance, "preview" n'est pas un suffixe officiel, cf section "Versioning" de https://github.com/KhiopsML/khiops/wiki/Release-Process)
Sinon, j'ai déjà ajouté un commentaire "update the following script arguments" dans le sens de la première approche.

File: packaging/windows/nsis/README.md (hunk 2 above)
Comment (marcboulle) [reply packaging/windows/nsis/README.md]: Done

File: packaging/windows/nsis/README.md (hunk 1 above)
Comment (popescu-v) [reply packaging/windows/nsis/README.md:46]: OK.
I would just add two comments, to make it clearer how the variables are assigned; something like:
1. "`KHIOPS_VERSION`: Khiops version for the installer. Should be identical to the value that is set in `KHIOPS_STR` in `src/Learning/KWUtils/KWKhiopsVersion.h`"
2. "`KHIOPS_REDUCED_VERSION`: Khiops version without suffix and only digits and periods. Thus, the pre-release fields of `KHIOPS_VERSION` are ignored in `KHIOPS_REDUCED_VERSION`.

Comment (marcboulle) [reply packaging/windows/nsis/README.md:46]: Done

Comment (popescu-v) [reply packaging/windows/nsis/README.md:46]: OK.

**Review by popescu-v — APPROVED** [review]
LGTM

**Pull request merged**